public/js/monitor.js

```diff
@@ -62,6 +62,35 @@ function doOauth(el, {emailWatch = false} = {}) {
     }
   });
 
+  // Growth Experiment: OAuth Entry Point IDs are unique to the experiment.
+  const oAuthEntryPointIds = [
+    "fx-monitor-check-for-breaches-blue-btn",
```

Comment:
maybe we should add "exp" to these so that it's easier later to sort these out.

Reply:
Marking this as an issue for next time. We're using these IDs in amplitude already, but we can make this change in future experiments.

Reply:
Filed https://github.com/mozilla/blurts-server/issues/1737

Reply:
sounds good but if they're already being used it may just cause more grief to switch them up now in any case.